app/components/article-featured-video.js

```diff
@@ -99,6 +113,41 @@ export default Component.extend(JWPlayerMixin, RespondsToScroll, {
 		}
 	},
 
+
+	onTouchStart(evt) {
+		if (!this.get('isOnScrollActive')) {
+			return;
+		}
+
+		const close = this.get('scrollCloseButton');
+		const bounds = close.getBoundingClientRect();
+		const eventX = evt.targetTouches[0].pageX;
+		const eventY = evt.targetTouches[0].pageY;
+		const distanceFromTheTopOfTheViewport = bounds.top + window.scrollY;
+
+		if (
+			close &&
+			eventX > bounds.left &&
+			eventX < bounds.right &&
+			eventY > distanceFromTheTopOfTheViewport &&
+			eventY < distanceFromTheTopOfTheViewport + close.offsetHeight
+		) {
+			this.set('wasCloseButtonClicked', true);
+			evt.stopPropagation();
+			this.send('dismissPlayer');
+		}
+	},
+
+	onTouchEnd(evt) {
+		if (this.get('wasCloseButtonClicked')) {
+			evt.preventDefault();
```

Comment:
`preventDefault` is not needed

Reply:
I checked it and it seems preventDefault needs to stay in order for the search not to open up after clicking the X button.